docs/docs/guides/fonts.md

```diff
@@ -0,0 +1,124 @@
+# Fonts
+
+## Loading Google Fonts with next-font, expo-font, and Tailwind/Nativewind
+
+This guide was written using the expo-router starter in mind. 
+
+1. Load fonts in expo, in apps/expo/_layout.tsx. Make sure to `npx expo add @expo-google-fonts/roboto-slab` whatever fonts you want to use.
+
+apps/expo/_layout.tsx
+```
+import { Provider } from 'app/provider';
+import { useFonts } from 'expo-font';
+import { Stack } from 'expo-router';
+import * as SplashScreen from 'expo-splash-screen';
+import { useEffect } from 'react';
+import { Gurajada_400Regular } from "@expo-google-fonts/gurajada";
```

Comment:
Let's get rid of any code that isn't directly related to this guide like SplashScreen etc.